### PR TITLE
Moving Import Paths and Adding `null` checks 

### PR DESCRIPTION
**Pull request by seanlowjk**
As mentioned in title.

### CODE REVIEW

File: src/components/Application.tsx [hunk 1]
```diff
@@ -3,20 +3,19 @@ import * as qs from 'query-string';
...
 import { Role, sourceLanguages } from '../reducers/states';
 import { stringParamToInt } from '../utils/paramParseHelpers';
-import { ExternalLibraryName, ExternalLibraryNames } from './assessment/assessmentShape';
-import Contributors from './contributors';
-import NavigationBar from './NavigationBar';
-import NotFound from './NotFound';
-
-import { Variant } from 'js-slang/dist/types';
 
 export interface IApplicationProps extends IDispatchProps, IStateProps, RouteComponentProps<{}> {}
```
Comment (anthony-halim) [inline src/components/Application.tsx:21]: Hi, please revert this change to the old codebase.

Comment (seanlowjk) [reply src/components/Application.tsx:21]: aight
